controllers/routes.go

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import (
+	"badi-cal/badi-events-backend/models"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type JSONTime time.Time
+
+func (t JSONTime) MarshalJSON() ([]byte, error) {
+	stamp := fmt.Sprintf("\"%s\"", time.Time(t).Format(time.RFC3339))
+	return []byte(stamp), nil
+}
+
+type App struct {
+	Models *models.Models
+}
+type Handler func(w http.ResponseWriter, r *http.Request, app *App)
+type HandlerDict map[string]Handler
+
+var Routes = map[string]HandlerDict{
+	"/notifications": {
+		http.MethodGet: NotificationsIndex,
+		http.MethodPut: NotificationsCreate,
```

Comment:
So at first glance I would say that this create API would have to be changed from a PUT method to a POST method as you are asking the API to create something and then it creates the unique identifier ID for the notification that was just created, but I also see that you are passing an `event_id` with the request payload so I guess that can also be an identifier for the notifications so you are asking the API to put this notification in the database with this `event_id`.  I am wondering to make that more clear if the path for creating a notification should actually be `/notifications/events/{event_id}`, but then maybe you might also need to enforce the `event_id` to be unique.  In that case if you do not want to enforce the `event_id` to be unique then you could leave the API path and maybe just change the PUT to a POST to make clear that the unique identifier for the notification is the ID that is created by the API.  Up to you.

Reply:
You're totally right, that should be POST. The event_id is coming from Google, we might want to use it in an update endpoint in the future.

Reply:
Fixed